src/common/options.cc

```diff
@@ -578,7 +583,10 @@ std::vector<Option> get_global_options() {
       "/usr/local/etc/ceph/keyring,/usr/local/etc/ceph/keyring.bin," 
   #endif
     )
-    .set_description(""),
+    .set_description("Path to a keyring file.")
+    .set_long_description("A keyring file is an INI-style formatted file where the section names are client or daemon names (e.g., 'osd.0') and each section contains a 'key' property with CephX authentication key as the value.")
+    .add_see_also("key")
```

Comment:
@jcsp i am wondering: if option `foo` "see_also"s option `bar`. does it imply that option `bar` also "see_also"s option `foo`. in short, is "see_also" considered symmetric?